example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
```

Comment:
No it's not, you're getting a different smaller result by the factor of the number of modes.